app/state_machines/workflows/conditionals.rb

```diff
@@ -11,4 +11,25 @@ def remove_response
       'awaiting_dispatch'
     end
   end
+
+  def unflag_for_clearance
```

Comment:
Can we give this a name that's more telling of it's function? e.g. `unflag_for_clearance_next_workflow`